### ISSUE TITLE
LE - Nemesis not compatible with xpmse (im aware of dummy fnis esp, but its still cause trouble)

### ISSUE THREAD

**Issue by kontoll123** [ShikyoKira/Project-New-Reign---Nemesis-Main#637]
i already heavy tested this. everytime i generate nemesis with xpmse installed, problem always occur (male enemy with one handed weapon hostile but wont attack, and movement stuck in places) i only use TUDM, and my workaround is i generate nemesis with tudm then make an output, then install xpmse. but as a result, xpmse cannot be read by some mods. hope this help, and addressed by author.

**Comment by kontoll123** [comment]
> I am experiencing the same problem same as u described on LE but it differs from mod to mod.. when I use TUDM reanimated + crouch sliding everything works perfectly. The trick for me was to place tudm above the crouch sliding in nemesis patch screen. Please try it. Also same for left and right panes of Mo2: TUDMre Crouch slide Crouchslidetudm FNIS
> 
> Then again if I bring hellbladetimed block in this setup some enemies can move and attack some cant, power attacking on spot but differs from race to race and eqıipped wep type. This can be due to I got different sized xpmse skeletons for each race. Even only cgo causing same bug for me including my player char this time, power attack and dodge in same spot. Never tried without xpmse but will try tho.
> 
> Edit: Removing xpmse cgo just works, enemies aswell. Gonna further test it without xpmse fixed scripts, UC and what not as UC Alone was also causing power attack on spot bug.

wait, are you using fnis with tudm?

**Comment by Suijja** [comment]
No..I mean the dummy plugin that Nemesis creates for XPMSE skeleton. -_-
